polaris-core/src/main/java/org/apache/polaris/core/storage/StorageLocation.java

```diff
@@ -44,8 +44,10 @@ protected StorageLocation(@Nonnull String location) {
       this.location = URI.create(location.replaceFirst("file:/+", LOCAL_PATH_PREFIX)).toString();
     } else if (location.startsWith("/")) {
       this.location = URI.create(location.replaceFirst("/+", LOCAL_PATH_PREFIX)).toString();
```

Comment:
Do we have tests for multiple leading `/` chars?

Reply:
No, we do not, but am introducing in revision. Thanks!